utils/remote.c

```diff
@@ -289,6 +289,7 @@ static tr_option opts[] =
     { 920, "session-info", "Show the session's details", "si", 0, NULL },
     { 921, "session-stats", "Show the session's statistics", "st", 0, NULL },
     { 'l', "list", "List all torrents", "l", 0, NULL },
+    { 'L', "labels", "Labels the current torrent(s)", "L",  1, "<label[,label...]>" },
```

Comment:
Slightly confusing phrasing since "Labels the current torrent" could be read as "set the current torrent's labels"

Maybe "List the current torrents' labels"?